### PR TITLE
Add DC doc

### LINKED ISSUE #438: K8SSAND-1291 ⁃ Document adding datacenter to an existing cluster
#21 introduced support for adding a DC to an existing cluster. We need to document what is involved with the process.

┆Issue is synchronized with this [Jira Task](https://k8ssandra.atlassian.net/browse/K8SSAND-1291) by [Unito](https://www.unito.io)
┆epic: K8ssandra Operator Documentation updates
┆fixVersions: k8ssandra-docs-v2
┆friendlyId: K8SSAND-1291
┆priority: Medium

### PR DESCRIPTION
**Pull request by jsanda**
**What this PR does**:
Provides documentation for adding a DC to an existing cluster

**Which issue(s) this PR fixes**:
Fixes #438

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [x] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [x] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Review by jeffbanks — CHANGES_REQUESTED** [review]

**Comment by johnsmartco** [comment]
John @jsanda, this doc is really helpful.  My thought with the current "docs-v2" Tasks section is to start over, replace all the topics currently deployed under https://docs-v2.k8ssandra.io/tasks/ ... (which are merely copies of the "docs-v1" content) ... with individual new/updated task-based topics like the one you wrote in this PR.  Over time, we'll add to the set of task-based articles under https://docs-v2.k8ssandra.io/tasks/<topic>.html.  Might categorize the tasks in doc subfolders when that makes sense.

File: docs/add-dc.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      arguments:
+        source_datacenter: dc1      
+```
+Cass Operator manages and reconciles CassandraTasks. A rebuild of all keyspaces will be performed on each node in `dc2`, one node at a time.
+
+The operator requeues reconciliation requests until the rebuild is finished
+
+**Note:** Upon successful completion Cass Operator deletes the CassandraTask.
+
+## Choose Source Datacenter for Streaming
```
[the remark this answers is not in the document]
Comment (jsanda) [reply docs/add-dc.md:166]: Good suggestion. I provide some detail.

File: docs/add-dc.md [hunk 2]
```diff
@@ -108,7 +114,9 @@ After `dc2` becomes ready the operator updates the replication strategy of *inte
 * `reaper_db`
 
 ## User Defined Keyspaces
-Next the operator updates the replication strategy of user-defined keyspsaces. The `k8ssandra.io/dc-replication` annotation must be set in order for the operator to update user-defined keyspaces. The value should be valid JSON. 
+Next the operator updates the replication strategy of user-defined keyspsaces. The
```
Comment (jeffbanks) [inline docs/add-dc.md:117]: mispelled keyspaces

**Review by jeffbanks — APPROVED** [review]
Looks like just a single misspelling.  Good to go after that!

**Pull request merged**